assets/js/modules/adsense/components/setup/SetupAccountNoClient.js

```diff
@@ -46,14 +45,13 @@ export default function SetupAccountNoClient() {
 				<Link
 					href="https://support.google.com/adsense/answer/6023158"
 					external
-					dangerouslySetInnerHTML={ sanitizeHTML(
-						__( 'Learn more<span class="screen-reader-text"> about updating your AdSense account</span>', 'google-site-kit' ),
-						{
-							ALLOWED_TAGS: [ 'span' ],
-							ALLOWED_ATTR: [ 'class' ],
-						}
-					) }
-				/>
+				>
+					{ __( 'Learn more', 'google-site-kit' ) }
+					{ ' ' }
+					<span className="screen-reader-text">
+						{ __( 'about updating your AdSense account', 'google-site-kit' ) }
+					</span>
```

Comment:
See above, this also shouldn't be split.

Reply:
Updated to use `createInterpolateElement`